Jenkinsfile.docker

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env groovy
+
+timeout(90) {
+node('shakedown') {
+  wrap([$class: 'MesosSingleUseSlave']) {
+  wrap([$class: 'AnsiColorBuildWrapper', 'colorMapName': 'XTerm']) {
+    def timeout=360000
+
+    properties([
+        parameters([
+          string(name: 'dcos_url',
+                defaultValue: '',
+                description: 'URL to DC/OS Enterprise Cluster'
+                ),
+
+          string(name: 'marathon',
+                defaultValue: '1.4.6',
+                description: '''Version of Marathon (MoM).  This is the version tag of the docker image.
```

Comment:
comment originally by @zen-dog "I would not exclude v here. Instead of passing the version, I would pass the tag here which might include v but also might look completely different in the future e.g. 1.5.0-SNAPSHOT-470-g660a7bd tags."